tests/baselines/reference/objectSpreadNegative.errors.txt

```diff
@@ -116,13 +114,9 @@ tests/cases/conformance/types/spread/objectSpreadNegative.ts(65,14): error TS269
     // generics
     function f<T, U>(t: T, u: U) {

```

Comment:
can you remove these two examples since objectSpread.ts now has a bunch more that pass.